CHANGELOG.md

```diff
@@ -1,5 +1,58 @@
 # Change Log
 
+## 2.13
+
+### Features
+
+- Implement Shader Model 6.8 min-spec
+  - SV_BaseInstance
+  - SV_BaseVertex
```

Comment:
These are actually called SV_Start{Index,Vertex}Location in D3D.

Reply:
Fixed.